src/Makefile

```diff
@@ -139,6 +139,9 @@ ifeq ($(uname_S),SunOS)
 	INSTALL=cp -pf
 	FINAL_CFLAGS+= -D__EXTENSIONS__ -D_XPG6
 	FINAL_LIBS+= -ldl -lnsl -lsocket -lresolv -lpthread -lrt
+	ifeq ($(USE_BACKTRACE),yes)
+	    FINAL_CFLAGS+= -DUSE_BACKTRACE
+        endif
```

Comment:
there's some odd mix of tabs and spaces in this file.
let's at least make the `endif` use the same ones as its `ifeq`